snuba/query/allocation_policies/bytes_scanned_rejecting_policy.py

```diff
@@ -38,7 +38,7 @@
     get_redis_client(RedisClientKey.RATE_LIMITER)
 )
 DEFAULT_OVERRIDE_LIMIT = -1
-DEFAULT_BYTES_SCANNED_LIMIT = 10000000
+DEFAULT_BYTES_SCANNED_LIMIT = 640000000000
```

Comment:
Reading the value is hard since its so big. How about this instead?
```suggestion
GIGABYTE = 10 ** 9
DEFAULT_BYTES_SCANNED_LIMIT = 640 * GIGABYTE
```